task/flake8/0.1/flake8.yaml

```diff
@@ -0,0 +1,46 @@
+---
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: flake8
+  labels:
+    app.kubernetes.io/version: "0.1"
+  annotations:
+    tekton.dev/pipelines.minVersion: "0.12.1"
+    tekton.dev/tags: python, flake8
+    tekton.dev/displayName: flake8
```

Comment:
can we add one more annotation for it `tekton.dev/categories` ? Probably you can refer the PR #658

Reply:
Done